src/pages/how-to/routing-traffic-to-private-networks.mdx

```diff
@@ -49,15 +51,49 @@ Nodes connected to routing peers will choose one of them to route packets to ext
 Masquerade hides other NetBird network IPs behind the routing peer local address when accessing the target Network range. This option allows access to your private networks without configuring routes on your local routers or other devices.
 
 If you don't enable this option, you must configure a route to your NetBird network in your external network infrastructure.
+
+### DNS Routes
+An alternative to specifying a network range directly is to use DNS routes. Instead of adding the network directly, you can add multiple domains in a route that will be dynamically resolved on the client. The resolved IP addresses for these domains will be added as routes.
+
+The interval for resolving DNS routes is 60 seconds by default, but you can change it using the `--dns-router-interval` flag:
```

Comment:
By default, DNS routes are resolved every 60 seconds. You can adjust this interval using the